src/Styleguide/Components/__stories__/SelectedExhibitions.story.tsx

```diff
@@ -15,50 +16,61 @@ const props = {
 
 storiesOf("Styleguide/Components", module).add("SelectedExhibitions", () => {
   return (
-    <React.Fragment>
-      <Section title="3 exhibitions responsive">
-        <SelectedExhibitions {...props} />
-      </Section>
-      <Section title="2 exhibitions responsive">
-        <SelectedExhibitions
-          exhibitions={exhibitions.slice(1) as any}
-          artistID="andy-warhol"
-          totalExhibitions={100}
-        />
-      </Section>
-      <Section title="3 exhibitions full">
-        <SelectedExhibitionsContainer {...props} />
-      </Section>
-      <Section title="2 exhibitions full">
-        <SelectedExhibitionsContainer
-          exhibitions={exhibitions.slice(1) as any}
-          artistID="andy-warhol"
-          totalExhibitions={100}
-        />
-      </Section>
-      <Section title="3 exhibitions collapsed (xs)">
-        <SelectedExhibitionsContainer
-          collapsible
-          exhibitions={exhibitions as any}
-          artistID="andy-warhol"
-          totalExhibitions={100}
-        />
-      </Section>
-      <Section title="2 exhibitions collapsed (xs)">
-        <SelectedExhibitionsContainer
-          collapsible
-          exhibitions={exhibitions.slice(1) as any}
-          artistID="andy-warhol"
-          totalExhibitions={100}
-        />
-      </Section>
-      <Section title="1 exhibition">
-        <SelectedExhibitions
-          exhibitions={exhibitions.slice(2) as any}
-          artistID="andy-warhol"
-          totalExhibitions={100}
-        />
-      </Section>
-    </React.Fragment>
+    <StorybooksRouter
+      routes={[
+        {
+          path: "*",
```

Comment:
SectionExhibitions uses the router `<PreloadLink>` internally. By setting a wildcard here it catches clicked links so that things don't transition weirdly in storybooks.

Reply:
Cool, this makes me think that functionality (linking to the artist cv page) should probably ideally live outside `SelectedExhibitions`, so it can be reused outside of the artist page context.